src/components/projects/wbs-details/work-package-container/work-package-details/work-package-details.test.tsx

```diff
@@ -35,6 +35,9 @@ describe('Rendering Work Package Details Component', () => {
       screen.getByText(`${endDatePipe(wp.startDate, wp.duration)}`, { exact: false })
     ).toBeInTheDocument();
     expect(screen.getByText(`${wp.progress}%`, { exact: false })).toBeInTheDocument();
+    expect(
```

Comment:
Not sure exactly why, but looking for what you suggest doesn't work out. 

![image](https://user-images.githubusercontent.com/59621104/157555527-ab928931-1874-493d-b49c-d03bdd501a63.png)

Reply:
Could you show us the entire html? Why is it not?

Reply:
This is the part it would show up. Maybe there's a way to compare to this directly?
![image](https://user-images.githubusercontent.com/59621104/157563061-2f6edf12-2be2-4cc2-902f-ee1eab19f1ab.png)

Reply:
Lol i forgot it was bold so that's probably making be that way